lib/mutant/mutator/node/arguments.rb

```diff
@@ -79,12 +74,11 @@ def emit_mlhs_expansion
           end
         end
 
-        # Return mlhs childs
+        # The mlhs childs
```

Comment:
What does mlhs stand for? If it's an uncommon abbreviation I would generally expand it in the documentation.